utils/apiLog.js

```diff
@@ -31,8 +31,9 @@ function getSize(obj) {
  * @param {Object} resultObj - API result object
  * @param {Object} logObject - from the request object
  * @param {Object or Array} retval - the returned object
+ * @param {Integer} customRecordCount - a custom recordCount
```

Comment:
pls. rename this to "recordCountOverride" and call it out as optional in the field description in the @param comments